src/main/java/org/jenkinsci/remoting/protocol/IOHub.java

```diff
@@ -140,6 +153,10 @@ private IOHub(Executor executor) throws IOException {
     public static IOHub create(Executor executor) throws IOException {
         IOHub result = new IOHub(executor);
         executor.execute(result);
+        if (Launcher.isWindows()) {
```

Comment:
BTW - I think this is no longer correct.

The network stack in my experience behaves the same when run on the Linux inside windows (ubuntu on bash on windows, for one example) - so checking for the file separator character is no longer correct.

there are some things that you want to do because you are on windows like paths, using different programes,  yet there are some things that you want to do due to differences in a protocol stack (like here)

Reply:
SO having run the tests on "bash on Ubuntu on Windows" org.jenkinsci.remoting.protocol.impl.NetworkLayerTest is hanging.
alas I can not get a stack trace from surefire when using this environment :(